### DISCUSSION TITLE
ZigBee repeater

### DISCUSSION THREAD

**Discussion by MTalon63** [ptvoinfo/zigbee-configurable-firmware#302] [Q&A] [answered]
Hi!
I'm curious if this firmware supports zigbee message repeat because as far as I know, every hardwired device should have that capability. I was planning on using CC2530 dev boards as my temperature sensors with stable 5V brick connected to the wall and acting as repeaters.

**Comment by ptvoinfo** [comment, marked as answer]
A repeater is named as "Router" in ZigBee. The firmware allows you to configure all devices as routers, but the router firmware for CC2530 is limited. Some sensors are excluded.

Please note, that CC2530 requires 3.3V
